frontend/packages/gitops-plugin/integration-tests/reporter-config.json

```diff
@@ -0,0 +1,14 @@
+{
+  "reporterEnabled": "mocha-junit-reporter, mochawesome",
+  "mochaJunitReporterReporterOptions": {
+    "mochaFile": "../../../gui_test_screenshots/junit_cypress-[hash].xml",
+    "toConsole": false
+  },
+  "mochawesomeReporterOptions": {
+    "reportDir": "../../../gui_test_screenshots/gitops-plugin/",
```

Comment:
This should just be `../../../gui_test_screenshots/`

Reply:
Removed all reporter-config.json files from all plugin folders and configured cypress.json accordingly